src/main/java/hudson/plugins/testlink/AbstractTestLinkBuilder.java

```diff
@@ -361,4 +371,26 @@ public Action getProjectAction(AbstractProject<?, ?> project) {
         return customFieldNamesArray;
     }
 
+
+    protected String[] createArrayOfTestPlanCustomFieldsNames(final VariableResolver<String> variableResolver,
```

Comment:
Great idea, and again, thanks for not doing that in this commit. It made it waaaay easier to review the code :grin: